src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseStatsRequest.tsx

```diff
@@ -282,17 +291,19 @@ class ReleaseStatsRequest extends React.Component<Props, State> {
       data: [],
     };
 
-    responseData.forEach(entry => {
-      const [timeframe, values] = entry;
-      const date = timeframe * 1000;
-      chartData.data.push({name: date, value: Math.round(values.duration_p50)});
-    });
-
     const sessionDurationAverage = Math.round(
-      meanBy(
-        chartData.data.filter(item => defined(item.value)),
-        'value'
-      )
+      mean(
+        responseData
+          .map(([timeframe, values]) => {
+            chartData.data.push({
+              name: timeframe * 1000,
+              value: Math.round(values.duration_p50),
+            });
+
+            return values.duration_p50;
+          })
+          .filter(duration => defined(duration))
```

Comment:
Is eliding data points going to skew the mean?

Reply:
Yes, we don’t want datapoints without duration to affect total average.